hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirectory.java

```diff
@@ -1591,8 +1591,8 @@ void moveInodes() throws IOException {
     }
 
     if (count != totalInodes) {
-      String msg = String.format("moveInodes: expected to move %l inodes, " +
-          "but moved %l inodes", totalInodes, count);
+      String msg = String.format("moveInodes: expected to move %d inodes, " +
```

Comment:
Hi,

Why do we change from %l to %d? Both variables are defined as long, not int.

Reply:
Only %d (Decimal integers) & %f (Decimal numbers) are supported. %l is not supported format, which lead to RTE.